app/views/collections/form.html.erb

```diff
@@ -45,6 +46,13 @@
       <% end %>
     <% end %>
 
+    <% component.with_pane do %>
+      <%= render Collections::Edit::PaneComponent.new(tab_name: :license, label: I18n.t('collections.edit.panes.license.label')) do %>
+        <p><%= t('collections.edit.panes.license.help_text') %></p>
```

Comment:
If this is only used here, put inline instead of in en.yml.

Reply:
Let's just leave as is. On the other hand, I don't want large blocks of text in form.html.erb either.